script/Deploy.s.sol

```diff
@@ -2,36 +2,53 @@
 
 pragma solidity 0.8.23;
 
-import { DeployBase } from "./DeployBase.s.sol";
+import { Script, console2 } from "../lib/forge-std/src/Script.sol";
 
-contract Deploy is DeployBase {
+import { IRegistrar } from "../src/interfaces/IRegistrar.sol";
+
+import { DeployBase } from "./DeployBase.sol";
+
+contract Deploy is Script, DeployBase {
     uint256 internal constant _STANDARD_PROPOSAL_FEE = 1e18; // 1 WETH
 
-    // NOTE: Ensure these are the correct cash token addresses.
+    // NOTE: Populate this arrays with cash token addresses.
     address[] internal _ALLOWED_CASH_TOKENS = [
         address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2) // mainnet WETH
     ];
 
-    // NOTE: Populate these arrays with accounts and starting balances.
-    address[] _initialPowerAccounts = [address(0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD)];
```

Comment:
those still need to be different lists and different balances

Reply:
I just saw that it was changed to double array, but it was very clean and nice before. what is the gain of change?

Reply:
It is to merge Power and Zero accounts into one array. Same for balances.
Would be nice to add a comment about which nested array is the Power one and which one is the Zero one.

Reply:
Stack too deep issues makes having separate arrays pretty much impossible. I will add comments.